modules/core/role/farm_role.managed_role_permissions.yml

```diff
@@ -4,5 +4,4 @@ farm_role:
     - access user profiles
     - change own username
   config_permissions:
-    - access administration pages
```

Comment:
Are we sure the only reason this permission was granted was for getting to the taxonomy lists?

Reply:
Yes I do think so! It really only enabled the Administration menu. I've done some testing and all the taxonomy term pages still work without the permission